### DISCUSSION TITLE
Multilanguage osu tutorial

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14549] [Ideas] [answered]
I propose to make a multilanguage osu tutorial map based on [current pishifat's](https://osu.ppy.sh/beatmapsets/1011011#osu/2116202) using difficulties as [country codes](https://en.wikipedia.org/wiki/Country_code).

**Comment by gagahpangeran** [comment, marked as answer]
This request is not related to the game itself if it's about the content of the map.

I think you should tell this idea to mapping community to create tutorial map with multi language narration and/or storyboard.

**Reply by Loreos7** [reply]
Since this map is pre-installed, I thought that such decisions are made by ppy (storyboard + voice acting)
And if ppy decides to do this, he tells pishifat or another guy to make ready-to-go map. 
IMHO that kind of map should be in osu for at least for 10 most popular langs.
Better to make a new map with a lazer's design storyboard which contains basics and some UI explanations (like score meter).

**Reply by gagahpangeran** [reply]
AFAIK, that map is community effort and initially start by community, then staff bundled it to osu!

And as I said before, you should tell this idea to mapping community instead.

**Reply by peppy** [reply]
yep, if you make this happen we will definitely consider it for inclusion!

**Reply by uzervlad** [reply]
I personally believe that there should be a standalone tutorial, rather than a tutorial map

**Reply by PercyDan54** [reply]
There's already things like Chinese Simplified edition:
https://osu.ppy.sh/b/2747713
